pkg/uidmap/uidmap.go

```diff
@@ -0,0 +1,136 @@
+package uidmap
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"syscall"
+
+	"github.com/docker/libcontainer/user"
+)
+
+func xlateOneFile(path string, finfo os.FileInfo, containerRoot uint32, inverse bool) error {
+	uid := uint32(finfo.Sys().(*syscall.Stat_t).Uid)
+	gid := uint32(finfo.Sys().(*syscall.Stat_t).Gid)
+	mode := finfo.Mode()
+
+	if (uid == containerRoot || gid == containerRoot) && !inverse {
```

Comment:
Why is this an issue? Does this imply this is partly owned by docker-root? A previously incomplete translation?